ROOTFS/etc/waggle/sanity/fatal/media_status.test

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+# Written by Kenan, yell at him if this breaks
+pf="Media status test"
+echo "$pf: Starting"
+
+# nx core tests
+
```

Comment:
These tests are all really good, but I think it'd be better to separate them to get more information in cases where we have more than one error